integration-tests/test/rnode.py

```diff
@@ -165,7 +166,7 @@ def extract_block_hash_from_propose_output(propose_output: str) -> str:
     return match.group(1)
 
 
-class Node:
+class Node:  # pylint: disable=R0904
```

Comment:
Is this `too-many-public-methods (R0904)`?  This class has just one public method.

Reply:
Yes . It is `too-many-public-methods`. 
All the method not started with `_` is public methods in `Node`. You must count it wrong.

Reply:
How many are there in your opinion?

Reply:
^

Reply:
26? What do you want to do with it?

Reply:
Oh, I got fooled by GitHub's folding of code 🤦‍♂ 

So I have 2 remarks: First, let's please not use crypting codes like R0904 but full names instead like I do everywhere else.  Second, in this case, the warning doesn't seem that useful so let's put it in `pylintrc`: https://github.com/rchain/rchain/blob/fbddb0d04573267159411c439e58bc0410003b7a/integration-tests/pylintrc#L55